client/src/test/java/org/apache/rocketmq/client/ValidatorsTest.java

```diff
@@ -58,7 +58,7 @@ public void testCheckTopic_HasIllegalCharacters() {
             Validators.checkTopic(illegalTopic);
             failBecauseExceptionWasNotThrown(MQClientException.class);
         } catch (MQClientException e) {
-            assertThat(e).hasMessageStartingWith(String.format("The specified topic[%s] contains illegal characters, allowing only %s", illegalTopic, "^[%|a-zA-Z0-9_-]+$"));
```

Comment:
Do we need to maintain the previous remark so that users can better know which topic does not comply with the rules?

Reply:
that is better